packages/dd-trace/src/debugger/devtools_client/index.js

```diff
@@ -107,9 +99,22 @@ session.on('Debugger.paused', async ({ params }) => {
         maxLength = highestOrUndefined(probe.capture.maxLength, maxLength)
       }
 
+      if (shouldVerifyConditions && probe.condition !== undefined) {
+        const { result } = await session.post('Debugger.evaluateOnCallFrame', {
+          callFrameId: params.callFrames[0].callFrameId,
+          expression: probe.condition,
+          returnByValue: true
+        })
+        if (result.value !== true) continue
```

Comment:
can we have a condition that fails with exception?

the spec should generate an evaluation errors every X minutes - so the user can fix those errors.
I think at the current state failed condition simply stop all probes from been trigger, is that right?

Reply:
Yes, that's an oversight from me when I created the condition logic. We should report any errors that might occur here. I'll do that in a follow up PR since it's nothing to do with log templates